Yoyo2Code/ReactIdeaBox
# Drag and Drop feature

What's this PR do?
This PR shows the code which allows for drag and drop.

* Where should the reviewer start?

Start first in idea_card.js file to see what functionality the card has. Then work your way up to the backlog and current lists. Both hold most of the functionality.

* How should this be manually tested?

This is manually tested visually to see if the components are rendering to the page without errors in console.

* Any background context you want to provide?

This application was created over the weekend starting from nothing. It uses an external api and prioritized functionality over code organization. The application allows drag and drop, but for some reason seems to work on the second time moving the card. Also, the arrow isn't functioning property as I wanted.  The window should also be zoomed out to at least 50%. The css displayed something completely differently and isn't changing even when reverting to older commits. So it might have had to do something with the browser. ***Card registers drag functionality best when dragged to bottom-middle white spacing inside of target card***

@carmer
@neight-allen
